app/services/scheduler.py

```
@@ -451,33 +457,89 @@ def check_storage_permissions_job_func() -> None:
                     location.permissions_error = None
+                    continue
+                if backend.capabilities().supports_local_path_stats:
+                    missing = _check_path_permissions(location.path)
+                    if missing:
+                        error = (
+                            f"Missing {' and '.join(missing)} permission on cold storage path: {location.path}"
+                        )
+                        if location.permissions_error != error:
+                            location.permissions_error = error
+                            logger.warning(
+                                f"Permission issue on cold storage '{location.name}': {error}"
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)